### DISCUSSION TITLE
Docker: Passport keys aren't created during docker setup

### DISCUSSION THREAD

**Discussion by cdn0x12** [pixelfed/pixelfed#5147] [Support]
I need to run `php artisan passport:install` manually after setting up the docker containers. 

If not, I'll get an error saying that `Invalid key supplied at /var/www/vendor/league/oauth2-server/src/CryptKey.php  line 67` and the timeline will be broken.

**Comment by jippi** [comment]
@CDN18 did you disable the `11-first-time-setup` script or have `DOCKER_APP_RUN_ONE_TIME_SETUP_TASKS=0` ? 

They should be created if `OAUTH_ENABLED` is `true` and the `first-time-setup` script is allowed to run

**Comment by Sh4d** [comment]
It seems that `OAUTH_ENABLED` is false by default, so just doing a deploy with all defaults is broken. Enabling OAUTH resolves the issue.
